deno/lib/types.ts

```diff
@@ -1114,8 +1119,30 @@ export class ZodString extends ZodType<string, ZodStringDef, string> {
   uuid(message?: errorUtil.ErrMessage) {
     return this._addCheck({ kind: "uuid", ...errorUtil.errToObj(message) });
   }
-  nanoid(message?: errorUtil.ErrMessage) {
-    return this._addCheck({ kind: "nanoid", ...errorUtil.errToObj(message) });
+  nanoid(
+    arg1?: number | string | { length?: number; message?: string },
```

Comment:
Accepting an object may be a good idea, but it's not implemented it so let's remove the type
```suggestion
    arg1?: number | string
```